OsmAnd/src/net/osmand/plus/settings/preferences/SizePreference.java

```diff
@@ -100,7 +106,11 @@ public CharSequence getSummary() {
 		return summary;
 	}
 
-	public String getValue() {
+	private boolean isPersistedStringEqualsZero(String persistedString) {
```

Comment:
1. Doube.parseDouble() = 0.0d is the same and even simpler
2. Just FYI BigDecimal.ZERO